src/backend/Dockerfile

```diff
@@ -4,7 +4,16 @@ RUN mkdir -p /usr/local/src/backend
 WORKDIR /usr/local/src/backend
 
 COPY requirements.txt /usr/local/src/backend/
-RUN pip install --no-cache-dir -r requirements.txt
+RUN set -ex \
```

Comment:
I am not sure why these were removed in previous commit, since I think the postgres client should be needed for django to work with a postgres DB container.

Reply:
Alright, I removed them locally and it seems fine. Lemme remove these changes!